frontend/resourceadm/components/NewResourceModal/NewResourceModal.tsx

```diff
@@ -3,30 +3,68 @@ import classes from './NewResourceModal.module.css';
 import { Button } from '@digdir/design-system-react';
 import { Modal } from '../Modal';
 import { ResourceNameAndId } from '../ResourceNameAndId';
+import { useCreateResourceMutation } from 'resourceadm/hooks/mutations';
+import { useNavigate, useParams } from 'react-router-dom';
+import { NewResourceType } from 'resourceadm/types/global';
+import { getResourcePageURL } from 'resourceadm/utils/urlUtils';
 
 interface Props {
   isOpen: boolean;
   onClose: () => void;
-  onCreateNewResource: (id: string, title: string) => void;
 }
 
 /**
  * Displays the modal telling the user that there is a merge conflict
  *
  * @param props.isOpen boolean for if the modal is open or not
  * @param props.onClose function to close the modal
- * @param props.onCreateNewResource function that handles the creation of a new resource
  */
-export const NewResourceModal = ({ isOpen, onClose, onCreateNewResource }: Props) => {
+export const NewResourceModal = ({ isOpen, onClose }: Props) => {
+  const navigate = useNavigate();
+
+  const { selectedContext } = useParams();
+  const repo = `${selectedContext}-resources`;
+
   const [id, setId] = useState('');
   const [title, setTitle] = useState('');
   const [editIdFieldOpen, setEditIdFieldOpen] = useState(false);
+  const [resourceIdExists, setResourceIdExists] = useState(false);
+  const [titleAndIdSame, setTitleAndIdSame] = useState(true);
```

Comment:
It's because of the logic of the page. The name might be missleading. 
initially the title and id is the same, so the user can type the title, and the ID updates automatically. The user will then have the option to edit the ID, and then the title and ID fields will not have the same values any more. The user can then save the edited ID, setting titleAndIdSame to false, or cancel the edited ID setting it back to true again. 

Changed the name to ```bothFieldsHasSameValue``` instead :)